db/const.txt

```diff
@@ -3431,3 +3431,14 @@ SEPTEMBER	9
 OCTOBER	10
 NOVEMBER	11
 DECEMBER	12
+
+// getpartymember( <party id>,<type> );
+PARTYMEMBER_NAME	0
+PARTYMEMBER_CID	1
+PARTYMEMBER_AID	2
+
+// getguildmember( <guild id>,<type> );
+GUILDMEMBER_NAME	0
+GUILDMEMBER_CID	1
+GUILDMEMBER_AID	2
```

Comment:
Cant they both share member name , cid and aid constants since the types are the same?